src/main/java/com/internet/shop/controller/InjectDataController.java

```diff
@@ -24,13 +25,29 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        userService.create(new User("admin", "admin", "1111", Set.of(Role.of("ADMIN"))));
-        userService.create(new User("User A", "userA", "1111", Set.of(Role.of("USER"))));
-        userService.create(new User("User B", "userB", "1111", Set.of(Role.of("USER"))));
-        userService.create(new User("User C", "userC", "1111", Set.of(Role.of("USER"))));
-        shoppingCartService.create(new ShoppingCart(3L));
-        shoppingCartService.create(new ShoppingCart(4L));
-        shoppingCartService.create(new ShoppingCart(5L));
+        User admin = new User("admin", "admin", "1111",
+                HashUtil.getSalt(), Set.of(Role.of("ADMIN")));
+        String password = HashUtil.hashPassword(admin.getPassword(), admin.getSalt());
```

Comment:
I agree with Dmytro - let's move this logic in UserServiceImpl class

Reply:
done